hdr-in-png-requirements.md

```diff
@@ -72,11 +74,7 @@ If the `iCCN` chunk is present, the image samples conform to the colour space re
 
 ### Decoder
 
-If the image contains a `cICP` chunk and will be rendered to a display or surface that supports `cICP`, then the PNG decoder shall ignore any `gAMA`, `cHRM`, `iCCP`, and `iCCN` chunks and use the `cICP` chunk instead. Otherwise, when a `iCCN` chunk is present, PNG decoders that recognize it and are capable of colour management shall ignore any `gAMA`, `cHRM`, and `cICP` chunks and use the `iCCN` chunk instead and interpret it according to [ICC] or [ICC-2010] as appropriate. PNG decoders that are used in an environment that is incapable of full-fledged colour management shall use the `gAMA` and `cHRM` chunks if present.
-
-#### Codestream
-
-A PNG datastream shall contain at most one embedded profile, whether specified explicitly with an `iCCP` or `iCCN` chunk or implicitly with an `sRGB` chunk.
+The `iCCN` chunk should be interpreted according to [ICC] or [ICC-2010] as appropriate. PNG decoders that are used in an environment that is incapable of full-fledged colour management shall use the `gAMA` and `cHRM` chunks if present.
```

Comment:
> PNG decoders that are used in an environment that is incapable of full-fledged colour management shall use the `gAMA` and `cHRM` chunks if present.

What is the objective of this clause? A decoder can also use `cICP` if present.

Reply:
I think the intent is that "incapable of full-fledged color management" means a true color profile cannot be upheld.

That said, maybe it would be best to spell out that a PNG decoder should go down the priority list until it finds a chunk that it can honor. Thoughts?

Reply:
> That said, maybe it would be best to spell out that a PNG decoder should go down the priority list until it finds a chunk that it can honor. Thoughts?

+1